src/vmm/src/resources.rs

```diff
@@ -1148,7 +1148,8 @@ mod tests {
                             "drive_id": "rootfs",
                             "path_on_host": "{}",
                             "is_root_device": true,
-                            "is_read_only": false
+                            "is_read_only": false,
+                            "io_engine": "Sync"
```

Comment:
could you explain this effect? Will the virtio device start reporting `io_engine` now? Is it not a breaking change?

Reply:
virtio devices were always reporting `io_engine`. Basically when device is configured, `io_engine` is optional, but during configuration inside FC, it becomes the default value of `Sync`. So if config is requested the `io_engine` will always be something. There is a test in `test_api.py` that checks the output.